### ISSUE TITLE
Get handle on the instance of chart created from controller

### ISSUE THREAD

**Issue by sr-nu** [gonewandering/angles#42]
As the created instance is not added to root/controller scope, it is not possible to interact with the instance (eg: SelectedSegments on chart instance.) is difficult. Can the instances be added to the controller or Root scope?

**Comment by ta4ka** [comment]
+1

**Comment by 72gm** [comment]
yeah, some (nice) way of interacting would be good... sort of defeats the purpose to use Angular then have to write some non angular code to interact with it
